packages/neuron-ui/src/components/NervosDAO/index.tsx

```diff
@@ -239,8 +239,12 @@ const NervosDAO = () => {
   }, [])
 
   useEffect(() => {
-    if (BigInt(CKBToShannonFormatter(depositValue)) > maxDepositAmount) {
-      setDepositValue(shannonToCKBFormatter(`${maxDepositAmount}`, false, ''))
+    try {
+      if (BigInt(CKBToShannonFormatter(depositValue)) > maxDepositAmount) {
+        setDepositValue(shannonToCKBFormatter(`${maxDepositAmount}`, false, ''))
+      }
+    } catch (error) {
+      // ignore error
```

Comment:
It is recommended to explicitly state that this is to ignore errors when converting to BigInt.

Reply:
Do you mean to add this statement for the annotation?
`When the depositValue is invalid, it displays the error in the textField, but it will throw an exception when valid wheater it's big than the max deposit value, and when the depositValue is invalid, it's no need to set max depositValue.`

Reply:
Will we add more statement or merge this PR first

Reply:
Okay, I have added.